AllReadyApp/Web-App/AllReady/ViewModels/Home/Index.cs

```diff
@@ -1,11 +1,10 @@
 ﻿using System.Collections.Generic;
-using AllReady.ViewModels.Campaign;
 
 namespace AllReady.ViewModels.Home
 {
-    public class HomePageViewModel
+    public class Index
```

Comment:
@stevejgordon I agree on making the class say `ViewModel`... more specific is better. That being said, the class is located in a ViewModels directory, so there is still clarity to how the class should be used when looking at the solution, but adding `ViewModel` to the class name would be more specific when looking at the class outside the context of the solution.

Some of these refactorings are part of what I'd like to see on a larger scale in this project, where we're building a specific ViewModel around GET/POST action methods instead of dragging one big ViewModel with 50+ properties across ALL actions methods in a controller, or even worse, sharing ViewModels between controllers.